dot/parachain/overseer/types.go

```diff
@@ -3,18 +3,12 @@
 
 package overseer
 
-import (
-	"context"
-
-	parachaintypes "github.com/ChainSafe/gossamer/dot/parachain/types"
-)
-
 // Subsystem is an interface for subsystems to be registered with the overseer.
-type Subsystem interface {
-	// Run runs the subsystem.
-	Run(ctx context.Context, OverseerToSubSystem chan any, SubSystemToOverseer chan any)
-	Name() parachaintypes.SubSystemName
-	ProcessActiveLeavesUpdateSignal(parachaintypes.ActiveLeavesUpdateSignal) error
-	ProcessBlockFinalizedSignal(signal parachaintypes.BlockFinalizedSignal)
-	Stop()
-}
+//type Subsystem interface {
```

Comment:
looks like this file does not need to exists since it is just comments

Reply:
Removed.